tests/core/data/test_filter_sql_table_data_node.py

```diff
@@ -30,60 +29,6 @@ def __init__(self, foo=None, bar=None, *args, **kwargs):
 
 
 class TestFilterSQLTableDataNode:
-    __pandas_properties = [
```

Comment:
I don't get this. 
What are these properties about? Who use them?

Reply:
This is probably a copy-paste error. We have similar variable in other test files, but it's not needed here

Reply:
OK thx